Source/DTDLv2/RealEstateCore/Asset/Equipment/ICTEquipment/ICTEquipment.json

```diff
@@ -0,0 +1,41 @@
+{
+  "@id": "dtmi:org:w3id:rec:ICTEquipment;1",
+  "@type": "Interface",
+  "description": {
+    "en": "Equipment and devices that are part of a building's ICT infrastructure."
+  },
+  "displayName": {
+    "en": "ICTEquipment"
+  },
+  "contents": [
+    {
+      "@type": "Property",
+      "name": "heightRUs",
+      "displayName": {
+        "en": "Height (RUs)"
+      },
+      "schema": "integer",
+      "writable": true
+    },
+    {
+      "@type": "Property",
+      "name": "numberOfPorts",
+      "displayName": {
+        "en": "Number of Ports"
+      },
+      "schema": "integer",
+      "writable": true
+    },
+    {
+      "@type": "Property",
+      "name": "standard",
```

Comment:
Can we add a description to this Property?

Reply:
Agree, not self-explanatory, will add

Reply:
Fixed with new commit.